### ISSUE TITLE
Add cerberus temlates

### ISSUE THREAD

**Issue by danmindru** [danmindru/responsive-html-email-signature#80]
https://tedgoas.github.io/Cerberus/

**Comment by paskainos** [comment]
Potential conflict...?

> [CSS Inliners](https://www.cerberusemail.com/templates#h-H3_5)
>
> - Cerberus is supposed to be simple and should not require a dependency like a CSS inliner.
> - The placement of Cerberus’s CSS is already optimized. The CSS in the `<head>` is meant only for email clients that parse CSS in this location. It doesn’t need to be inlined.
> - There are some CSS selectors like :hover that don’t inline so well and cause errors in some CSS inliners.
> - Inlining leads to code bloat. Not only does this impact download speeds, but some email clients like Gmail and iOS Outlook truncate messages after they exceed a certain file size.

**Comment by paskainos** [comment]
@danmindru Agreed! We (and our clients) have benefited a great deal from your pioneering work. Please let me know if we can help with more templates. FWIW: Awhile back, we goofed around with [mailchimp/email-blueprints](https://github.com/mailchimp/Email-Blueprints). The integration was seamless, but using them proved (for us) clunky and prohibitive.
